### DISCUSSION TITLE
Missing Activities In Explore Feed (FIXED)

### DISCUSSION THREAD

**Discussion by Maxitendo1** [community/community#137959] [Mobile] [answered]
I only get followed, starred and recommendations In the discover tab, but I have everything checked.

FIX: Now the releases and everything is showing up for me by doing this fix:

Go to your browser https://github.com/ -> Home -> Filters -> Scroll down -> Check: Include events from starred repositories

![Screenshot_20240906-212753.png](https://github.com/user-attachments/assets/1f335215-06d7-4839-9d6e-257853af2f0a)

![Screenshot_20240906-213110.png](https://github.com/user-attachments/assets/75e09294-e3c3-478c-bae0-69918dbb0ed7)

**Comment by Iron-Ham** [comment, marked as answer]
Thanks for reaching out @Maxitendo1! It seems we don't currently support that toggle on the mobile clients. I'll open up an internal ticket with the iOS and Android teams to roll out support.

Thanks,
Hesham
